### PR TITLE
Restructure Prysm Wallet V2 Package to Prevent Import Cycles

### PR DESCRIPTION
**Pull request by rauljordan**
No tracking issue. This PR refactors the accounts v2 package by separating the wallet code into its own subpackage to prevent any import cycles. Given wallet does not depend on much else but the cli context, it is a good candidate for splitting into its own package.

### CODE REVIEW

**Comment by rauljordan** [comment]
@rkapka please take a look as this helps prevent import cycles in accounts-v2

File: validator/rpc/wallet_test.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/prysmaticlabs/prysm/shared/testutil/assert"
 	"github.com/prysmaticlabs/prysm/shared/testutil/require"
 	v2 "github.com/prysmaticlabs/prysm/validator/accounts/v2"
+	v22 "github.com/prysmaticlabs/prysm/validator/accounts/v2/wallet"
```
Comment (nisdas) [inline validator/rpc/wallet_test.go:16]: how about a better alias here, `v22` is pretty confusing here

File: validator/rpc/wallet.go [hunk 2]
```diff
@@ -7,14 +7,16 @@ import (
...
 	ptypes "github.com/gogo/protobuf/types"
 	"github.com/pkg/errors"
+	"github.com/tyler-smith/go-bip39"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
 	pb "github.com/prysmaticlabs/prysm/proto/validator/accounts/v2"
 	"github.com/prysmaticlabs/prysm/shared/rand"
 	v2 "github.com/prysmaticlabs/prysm/validator/accounts/v2"
+	v22 "github.com/prysmaticlabs/prysm/validator/accounts/v2/wallet"
```
Comment (nisdas) [inline validator/rpc/wallet.go:17]: same concerns here

File: validator/rpc/wallet.go [hunk 3]
```diff
@@ -7,14 +7,16 @@ import (
 
 	ptypes "github.com/gogo/protobuf/types"
 	"github.com/pkg/errors"
+	"github.com/tyler-smith/go-bip39"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
```
Comment (nisdas) [inline validator/rpc/wallet.go:13]: empty space

File: validator/accounts/v2/wallet_create.go [hunk 4]
```diff
@@ -3,30 +3,32 @@ package v2
 import (
 	"context"
 	"fmt"
-	"path/filepath"
 
 	"github.com/manifoldco/promptui"
 	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+
```
Comment (0xKiwi) [inline validator/accounts/v2/wallet_create.go:10]:
```suggestion
```

File: validator/node/node.go [hunk 5]
```diff
@@ -14,6 +14,9 @@ import (
 	"syscall"
 
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+
```
Comment (0xKiwi) [inline validator/node/node.go:19]:
```suggestion
```

File: validator/rpc/server.go [hunk 6]
```diff
@@ -9,19 +9,20 @@ import (
 	recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
 	grpc_opentracing "github.com/grpc-ecosystem/go-grpc-middleware/tracing/opentracing"
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+	"github.com/sirupsen/logrus"
+	"go.opencensus.io/plugin/ocgrpc"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/reflection"
+
```
Comment (0xKiwi) [inline validator/rpc/server.go:17]:
```suggestion
```

**Review by 0xKiwi — DISMISSED** [review]

**Review by 0xKiwi — APPROVED** [review]

**Pull request merged**